pmd-core/src/main/java/net/sourceforge/pmd/ant/Formatter.java

```diff
@@ -57,12 +61,26 @@ public Renderer getRenderer() {
     }
 
     public void start(String baseDir) {
+
+        Properties properties = createProperties();
+        
+        Charset charset;
+        {
+            String s = (String) properties.get("encoding");
+            if (null == s) {
+                charset = StandardCharsets.UTF_8;
+                properties.put("encoding", charset.name());
+            } else {
+                charset = Charset.forName(s);
+            }
+        }
+
         try {
             if (toConsole) {
-                writer = new BufferedWriter(new OutputStreamWriter(System.out));
+                writer = new BufferedWriter(new OutputStreamWriter(System.out, charset));
```

Comment:
> But sun.* properties are non standard and may not be present on different JREs...

I know, it's only best effort. Note that it could then default to `file.encoding` instead of UTF-8, which might be more sensible for console output.